deployment/mcad-controller/crds/mcad.ibm.com_appwrappers.yaml

```diff
@@ -828,3 +827,6 @@ spec:
         type: object
     served: true
     storage: true
+    subresources:
```

Comment:
@asm582 and @dmatch01 Does this change make sense to you? Why wasn't added earlier ?

Reply:
subresources are used for status and scaling, scaling AW does not make sense to me. status of AW is currently driven by pods spawned and represented in `Status` block. subresource itself is optional for any CRDs